src/chat/discord.md

```diff
@@ -1,4 +1,104 @@
 ## Discord
 
-We haven't yet written the documentation for Discord, but if you'd like to help
-out, this is the place for it to go!
+Rust's Discord is used by Community, Lang, Ops, and Documentation teams, as well as their working groups. It is also the home of Domain Working Groups and provides a space for general discussion among Rust users, contributors, and beginners. 
```

Comment:
Lang team has moved to Zulip.

Reply:
Phrasing should be oriented around the Discord being *maintained* for Rust collaboration, not that it is their fixed home.

Reply:
Revised to address these comments. Thanks!